test-framework/test-suites/integration/tests/add/test_add_host.py

```diff
@@ -0,0 +1,51 @@
+import pytest
```

Comment:
This file probably doesn't belong in this commit. I'm fine with checking in tests as we see the need/have the time, but they should go in their own commits if they aren't related.